### DISCUSSION TITLE
Improvement - Removed exception catch on failed Event persistence

### DISCUSSION THREAD

**Discussion by artur-baltabayev** [keycloak/keycloak#8567] [Archive]
This was done before for admin events in the AdminEventBuilder.
The issue was that events did not trigger a transaction rollback due to the exception being handled and converted to a simple log message. We discovered that there is another such place in the code and would want to correct that aswell.

PR : https://github.com/keycloak/keycloak/pull/8568

Here is the previous pr for the admin events : https://github.com/keycloak/keycloak/pull/8166

**Comment by stianst** [comment]
Makes sense and doesn't need discussing since we've already discussed this for admin events.
